### PR TITLE
Video tutorials

### PR DESCRIPTION
**Pull request by ralphilius**
There's some notes on this bounty:

- Should the award be variable based on the content and the quality?
- Where should we store the video? Inside this repo? If not then we need some storage controlled by @tyvdh just in case the submitted resource is made unavailable by submitters in the future.

### CODE REVIEW

File: bounties/level-2/video-tutorials.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Video tutorials
+
+## Award
+| Hunter | Reviewer | Type
+| :- | :- | :-
+| 750 XLM (+750 XLM for established creators) | 1000 XLM | Continuous
```
Comment (hanseartic) [inline bounties/level-2/video-tutorials.md:6]: I think creating high-quality appealing content is way more effort then judging the result. So even the basic hunter's bounty should be higher then the reward.

File: bounties/level-2/video-tutorials.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+### What are the requirements for the bounty hunter?
+
+ - The bounty hunter should have good knowledge of Stellar and can create productive walkthrough videos. They should be familiar with Stellar supported SDKs.
+ - `(Optional)` The bounty hunter could double the reward if they have a well-established viewer base such as:
+   - A Youtube channel with more than 1000 subscribers
+   - A tutorial blog with more than 10,000 pageviews per month
+
+### What are the deliverables?
+
```
Comment (hanseartic) [inline bounties/level-2/video-tutorials.md:31]:
```suggestion
- the video itself (more specifically a link to the video)
- if there was a coding example a link to the codebase - preferrably a branch or dedicated repo
```

File: bounties/level-2/video-tutorials.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+#### Review Criteria
+  - The quality of content in the video
+  - The quality of the code being presented 
+    - Since people would probably copy and paste the code while watching, the code quality should be checked
+    - Should avoid less secure and performant source code, unless stated during the walkthrough
+  
+#### Submission Procedure
+  - Create a work issue 
+  - Add a link to your video in the work issue
+  - Upload the final video on Youtube and create a PR to upload it to assets folder
```
Comment (hanseartic) [inline bounties/level-2/video-tutorials.md:45]: I'd rather have some place where we collect the links to the videos. Maybe as an article on or tag-collection medium.com.

File: bounties/level-2/video-tutorials.md (hunk 1 above)
Comment (ralphilius) [reply bounties/level-2/video-tutorials.md:6]: Agreed. I think the reviewer should need to watch the tutorial and spot any issues with it. Maybe we can increase the hunter award and reduce reviewer's.

File: bounties/level-2/video-tutorials.md (hunk 3 above)
Comment (ralphilius) [reply bounties/level-2/video-tutorials.md:46]: Maybe Youtube video with tags? Or when we have a website for bounties, we can gather there.

**Review by tyvdh — CHANGES_REQUESTED** [review]

**Pull request merged**